MySQL/system/ksp-restrict-access-mysqldump.yaml

```diff
@@ -0,0 +1,20 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/ 
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-restrict-access-to-mysqldump
+  namespace: default # Change your namespace
+spec:
+  tags: ["MySQL", "mysqldump"]
+  message: "Alert! mysqldump command is has been used."
+  selector:
+    matchLabels:
+      app: mysql # Change your labels
+  process:
+    severity: 10
+    matchPaths:
+    - path: /usr/bin/mysqldump
+    action: Block
```

Comment:
Is there any compliance/stig rule which says that mysqldump should not be allowed? Can we add that ref here?